test/index.html

```diff
@@ -139,6 +139,7 @@
 			<foo_bar id="foobar">test element</foo_bar>
 		</form>
 		<b id="floatTest">Float test.</b>
+		<div id="customPropertyTest">Custom property test</div>
```

Comment:
It is better to generate html code in the test itself otherwise it is hard to keep track of fixtures and tests